jboss-eap-clustered-multivm-rhel8/README.md

```diff
@@ -112,7 +111,7 @@ Build your environment with JBoss EAP 7.2 cluster on top of n number of RHEL 8.0
 
     - **RHSM Pool ID for RHEL OS** - Red Hat Subscription Manager Pool ID (Should have RHEL entitlement). This is mandatory when selecting BYOS RHEL OS as license type.  This should be left blank when selecting RHEL OS License type PAYG.
 
-    - **Storage Replication** - Choose the Replication Strategy for your Storage account.
+    - **Storage Replication** - Choose the [Replication Strategy](https://docs.microsoft.com/en-us/azure/storage/common/storage-redundancy) for your Storage account.
```

Comment:
Change accommodated - Under “Deployment Steps” in point about RHEL OS License Type